pulp_rpm/app/models/distribution.py

```diff
@@ -106,8 +102,63 @@ class Meta:
             "build_timestamp",
         )
 
+    def get_copy(self, repository):
```

Comment:
Any other ideas or better ways to create a copy are welcome

Reply:
I like this

Reply:
:+1: from me as well.